ci/config.yaml

```diff
@@ -8,9 +8,15 @@ validate-yaml: true
 validate-maintainers: false
 check-version-increment: true
 
-# Consider helm charts in all directories.
+# Consider helm charts in directories. '.' as chart-dir does not work.
 chart-dirs:
```

Comment:
Or maybe remove the explicit whitelist to have it run by default?

Exclusion (below) is always possible.

Reply:
Tested locally, we need the explicit list sadly, because of our folder structure.
See issues:
https://github.com/helm/chart-testing/issues/633
https://github.com/helm/chart-testing/issues/226